Formula/s/screenpipe.rb

```diff
@@ -0,0 +1,38 @@
+class Screenpipe < Formula
+  desc "Library to build personalized AI powered by what you've seen, said, or heard"
+  homepage "https://github.com/mediar-ai/screenpipe"
+  url "https://github.com/mediar-ai/screenpipe/archive/refs/tags/v0.1.98.tar.gz"
+  sha256 "cb3c8039ecb60d35bacd2b9673db112f907b4a1d3d7c32f49a5e77c0274268ad"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "rust" => :build
+  depends_on "ffmpeg"
+  depends_on :macos
+
+  on_macos do
+    def install
+      ENV["PKG_CONFIG_PATH"] = "/usr/local/opt/ffmpeg/lib/pkgconfig:#{ENV["PKG_CONFIG_PATH"]}" if Hardware::CPU.intel?
+      ENV["PKG_CONFIG_ALLOW_CROSS"] = "1" if Hardware::CPU.intel?
+
+      target_arch = Hardware::CPU.arm? ? "aarch64-apple-darwin" : "x86_64-apple-darwin"
```

Comment:
```suggestion
```
You don't need these.

Reply:
Because `brew` already sets the correct `PKG_CONFIG_*` flags, and the target arch is redundant because we're compiling for the host (i.e. we're not cross-compiling).

Reply:
Probably not, not. Why doesn't `cargo install` work?